internal/appsec/emitter/httpsec/options.go

```diff
@@ -0,0 +1,24 @@
+package httpsec
+
+import "net/http"
+
+type WrapHandlerCfg struct {
+	OnBlock            []func()
+	ResponseHdrFetcher func(http.ResponseWriter) http.Header
+}
+type WrapHandlerOption func(*WrapHandlerCfg)
+
+func defaultWrapHandlerCfg() *WrapHandlerCfg {
+	return &WrapHandlerCfg{
+		OnBlock: []func(){},
```

Comment:
I would personally have left the zero-value (`nil`) here... Unlike `map[K]V`, moving a `nil` slice around doesn't come with a way-too-easy-to-cause `nil` panic.